.golangci.yml

```diff
@@ -73,20 +66,20 @@ linters:
     - dupl
     - errcheck
     - exportloopref
-    - funlen
+    #- funlen
```

Comment:
Isn’t it better to just remove the line?

Reply:
The problem is that if I'll remove that line, I would never know that it's an option..